pwa/src/components/common/elementCreationNew.tsx

```diff
@@ -77,32 +77,42 @@ const ElementCreationNew: React.FC<ElementCreationNewProps> = ({id, label, data,
 
       <div className="row">
         <h5>Create {label}</h5>
-        <div className="col-10">
           {
             select ? (
```

Comment:
I think the styling issues started when this `select` option was added to this `ElementCreation` component. I think after the MVP we will be refactoring a lot, so for now we can leave this as is.

However, I think we shouldn't have added this to this component. It makes it more complex and hard to maintain and also hard to build upon.

Reply:
Note: this could've been its own component. 👍🏼